afs/afs-ws/afs-ws-server-utils/pom.xml

```diff
@@ -36,13 +36,22 @@
             <artifactId>javaee-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>io.jsonwebtoken</groupId>
+            <artifactId>jjwt</artifactId>
+        </dependency>
 
         <!-- Test dependencies -->
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>powsybl-afs-ws-utils</artifactId>
+            <version>${project.version}</version>
```

Comment:
Is scope test missing?

Reply:
No it just in the wrong section, moved to compile dependencies